programs/wen_new_standard/src/instructions/mint/create.rs

```diff
@@ -24,11 +19,10 @@ pub struct CreateMintAccountArgs {
     pub uri: String,
 }
 
-pub const MINT_EXTENSIONS: [ExtensionType; 4] = [
+pub const MINT_EXTENSIONS: [ExtensionType; 3] = [
     ExtensionType::MetadataPointer,
     ExtensionType::GroupMemberPointer,
     ExtensionType::TransferHook,
-    ExtensionType::MintCloseAuthority,
```

Comment:
why did you drop this?

Reply:
Will add it back in a different PR with burn, delegate and freeze